RELEASING.md

```diff
@@ -3,23 +3,27 @@
 1. checkout master
 2. pull from repo
 3. run the unittests
-4. run `loghub` and update the `CHANGELOG.md` file:
+4. run `loghub`.  Replace <github username> and <previous version> with proper
+   values.  To get the previous version run `git --tag` and select the most
+   recent with highest version number.
 
 ```
-loghub pytroll/satpy -u <username> -st v0.8.0 -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes" -plg backwards-incompatibility "Backwards incompatible changes"
+loghub pytroll/satpy -u <github username> -st v<previous version> -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes" -plg backwards-incompatibility "Backwards incompatible changes"
 ```
 
-Don't forget to commit!
+This command will create a CHANGELOG.temp file which need to be added
+to the top of the CHANGLOG.md file.  Remember to update also the
+version number to the same given in step 5. Don't forget to commit
+CHANGELOG.md!
```

Comment:
Does `CHANGELOG.temp` differ from the stdout of the command? I've never actually used/opened the temp file. I've just copied the output from the terminal.

Reply:
Didn't do comparison, but I guess they were identical. I'll mention this.